{{cookiecutter.project_slug}}/scripts/python/gitlab_sync.py

```diff
@@ -49,9 +49,26 @@ def get_group(self):
 
     def create_group(self):
         """Create a GitLab group and sub-projects with badges."""
-        self.group = self.gl.groups.create(
-            {"name": self.project_name, "path": self.group_slug}
-        )
+        groups = self.gl.groups.list(search=self.project_name)
+        try:
+            self.group = groups[0]
+        except IndexError:
+            while len(groups) < 1:
+                create_group = input(
+                    'Create a group on gitlab.com then press enter to go ahead. Type "exit" to close this setup: '  # noqa
```

Comment:
Split this line please:
```suggestion
                    "Create a group on gitlab.com then press enter to go ahead. "
                    "Type 'exit' to close this setup: "
```